lib/Controller/PollController.php

```diff
@@ -126,13 +127,85 @@ public function createPoll(string $question, array $options, int $resultMode, in
 
 		try {
 			$this->chatManager->addSystemMessage($this->room, $attendee->getActorType(), $attendee->getActorId(), $message, $this->timeFactory->getDateTime(), true);
-		} catch (\Exception $e) {
+		} catch (Exception $e) {
 			$this->logger->error($e->getMessage(), ['exception' => $e]);
 		}
 
 		return new DataResponse($this->renderPoll($poll), Http::STATUS_CREATED);
 	}
 
+	/**
+	 * Modify a draft poll
+	 *
+	 * Required capability: `edit-draft-poll`
+	 *
+	 * @param int $pollId The poll id
+	 * @param string $question Question of the poll
+	 * @param string[] $options Options of the poll
+	 * @psalm-param list<string> $options
+	 * @param 0|1 $resultMode Mode how the results will be shown
+	 * @psalm-param Poll::MODE_* $resultMode Mode how the results will be shown
+	 * @param int $maxVotes Number of maximum votes per voter
+	 * @return DataResponse<Http::STATUS_OK, TalkPollDraft, array{}>|DataResponse<Http::STATUS_BAD_REQUEST, array{error: 'draft'|'options'|'question'|'room'}, array{}>|DataResponse<Http::STATUS_NOT_FOUND, array{error: string}, array{}>
```

Comment:
Same here, instead of `array{error: string}` I'd prefer to explicitly mention the strings we know